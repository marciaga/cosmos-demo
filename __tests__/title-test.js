import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../src/atomic/title';

test('it renders a title component', () => {
  const tree = renderer.create(<Title />).toJSON();

  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule('font-size', '1em');
});
