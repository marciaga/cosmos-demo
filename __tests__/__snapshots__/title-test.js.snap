// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders a title component 1`] = `
.c0 {
  font-size: 1em;
  text-align: center;
  color: royalblue;
}

<h1
  className="c0"
/>
`;
